src/components/Modal/ModalContainer.tsx

```diff
@@ -8,23 +8,41 @@ import { Modal } from './Modal';
 export const ModalContainer: React.FC<ModalContainerProps> = ({
   onClose,
   addTodo,
+  editTodo,
+  currIDForEdit,
+  modalType,
+  initialValueForModalWithEditFeature,
 }) => {
-  const [title, handleTitle] = useInput();
-  const [description, handleDescription] = useInput();
+  const { title: initialTitle } = initialValueForModalWithEditFeature;
+  const { description: initialDescription } =
+    initialValueForModalWithEditFeature;
 
+  const [title, handleTitle] = useInput(
+    modalType === 'edit' ? initialTitle : ''
+  );
+  const [description, handleDescription] = useInput(
+    modalType === 'edit' ? initialDescription : ''
+  );
   const createTodo = () => {
     addTodo({ id: v4(), title: title, description: description });
 
     onClose();
   };
+
+  const onSubmit = () => {
+    if (modalType === 'create') createTodo();
+    else editTodo(currIDForEdit, title, description);
```

Comment:
I don't see the reason for using this id at all. Try to delete it.
Don't use this kind of short expression. Use curly braces.

Reply:
deleted, used curly braces